deployer/README.md

```diff
@@ -1175,3 +1175,262 @@ the first created. You can check if the route in question is defined in multiple
     logging     kibana-ops   kibana-ops.example.com              logging-kibana-ops
 
 (In this example there are no overlapping routes.)
+
+## Splitting your Fluentd for high scale capacity (Advanced)
+
+In scenarios where the number of nodes running Fluentd starts to exceed 100, the
+watching connection to Kubernetes that is used within Fluentd to collect pod
+metadata can cause the cluster to exceed the default value for the *maxRequestsInFlight*
+configuration setting of `400`. Fluentd uses two watches, one for pod
+changes and one for namespace changes. This means running Fluentd on around 200
+nodes, the cluster would start to experience symptoms such as Fluentd being unable
+to connect to the Kubernetes service to collect metadata and preventing it from
+being able to process and ship logs.
+
+The alternative to setting *maxRequestsInFlight* to be unlimited, potentially creating
+a larger load on the cluster, is to split up the Fluentd deployments into collector
+and enricher pods (as opposed to all-in-one which is provided OOTB).
+
+This configuration leverages Fluentd reading its configuration from a configmap
+and the Fluentd image having the [fluent-plugin-secure-forward](https://github.com/tagomoris/fluent-plugin-secure-forward)
+plugin installed.
+
+First, unschedule all of Fluentd pods (Elasticsearch, Kibana, and Curator
+can all remain running):
+
+    $ oc label node --all logging-infra-fluentd-
+
+Where "logging-infra-fluentd" is the nodeSelector configured for the Fluentd daemonset.
+
+Next, create the key/cert pair that will be used for each Fluentd to communicate
+with one another:
+
+    $ openssl req -new -newkey rsa:2048 -days 712 -x509 -keyout server.key -out server.crt -subj "/CN=logging-enrichment-fluentd/OU=OpenShift/O=Logging" -passout pass:kspass
+
+Once the cert and key are created, the logging-fluentd secret needs to add these
+generated files:
+
+    $ oc create secret generic logging-fluentd --dry-run=true --from-file server.crt --from-file server.key -o yaml | oc apply -f -
```

Comment:
I would put the `dry-run` flag next to the `-o yaml` flag ... first reaction was "oh, he left the --dry-run flag in by accident"